.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contribute to PurotoApp/Berry
+
+First of all, thank you for taking a moment to consider contributing to Puroto! You're awesome.
+
+> Before contributing, please read our [Code of Conduct](https://github.com/PurotoApp/Berry/blob/master/.github/CODE_OF_CONDUCT.md) first. We want this community to stay civil and respecting to each other.
+
+Contributing to Berry can be done in many ways, also in more than 'just' code. Available options you can consider are:
+
+- Implement a feature or fix a bug
+- Create issues on bugs and feature-requests
+- Fund the development
+- Spread the word!
+
+## Implement a feature or fix a bug
+
+Implementing a feature or fixing bugs can be done via "Pull-Requests". Doing that is really easy. If you are new to open source, feel free to check out one of the following resources:
+
+- https://opensource.guide/how-to-contribute/
+- https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request
+
+If you are a new to our project and first want to get comfortable with contributing, feel free to check out the list of [good first issues](https://github.com/PurotoApp/Berry/labels/good%20first%20issue). They're features and bugs, that should be easily implemented or fixed. A good way to get started!
+
+If you are already experienced in open source development, feel free to check out or open issues and feature request. If you have a brilliant idea for a new feature that great! Just please [open an issue](#Creating issues) with an explanation of what you are going to implement first to discuss with our staff how the feature should be implemented to enhance the project. Don't forget to add the issue to your PR later on!
+
+## Creating issues
+
+You found a bug or have an exciting idea for a new feature? Or just have a question or concern? Fell free to create an issue on our github repository, so we can resolve it as quickly as possible. If you are new to github, please see the [official github guide on how to create issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue).
```

Comment:
There's a typo to `Fell free to create ...`, you should probably replace `Fell` to `Feel`.